sdk/eventhub/Microsoft.Azure.EventHubs/ReadMe.md

```diff
@@ -25,7 +25,7 @@ Use the client library for Event Hubs to:
 
 - Receive events from one or more publishers, transform them to better meet the needs of your ecosystem, then publish the transformed events to a new stream for consumers to observe.
```

Comment:
I don't think it is a requirement (at least not yet :)) but for consistency most places name this file README.md in all uppercase. I know that was already the case before the move but given case sensitivity issues on the different OS's the best time to change the case of a file is part of a move like you are doing here.

Reply:
Done.   _(I'd agree that this is a good time for such things)_